### PR TITLE
Feature/redux statement

### PR DESCRIPTION
**Pull request by Ksenia-Tulikova**
- Added redux statement to my app
- Added Feedback page styles and html
- Created actions
- Played around dispatching an action and changing redux state 
- Added the possibility to save entered data to Local Storage
PREWIEW
Feedback
: 

PLANS:
- add validations to my inputs

### CODE REVIEW

File: client/src/modules/Home/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 import { Home } from './components/Home';
+import { connect } from 'react-redux';
 
-export default Home;
+export default connect()(Home);
```
Comment (AV29) [inline client/src/modules/Home/index.js]: Do you need to connect to Home?

File: client/src/modules/Home/reducers/homeReducer.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const INITIAL_STATE = {};
+
+const homeReducer = (state = INITIAL_STATE, action) => {
```
Comment (AV29) [inline client/src/modules/Home/reducers/homeReducer.js]: same here...looks like a reducer mock

File: client/src/modules/WorldWeather/reducers/worldWeatherReducer.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+const INITIAL_STATE = {
+  title: 'World Weather Searcher'
+};
+
+export function worldWeatherReducer(state = INITIAL_STATE, action) {
+  switch (action.type) {
+    case 'ActionNAME': {
```
Comment (AV29) [inline client/src/modules/WorldWeather/reducers/worldWeatherReducer.js]: and here

File: client/src/modules/Home/reducers/homeReducer.js (hunk 2 above)
Comment (Ksenia-Tulikova) [reply client/src/modules/Home/reducers/homeReducer.js]: Yeah it was a reducer plug
I've already deleted all this unnecessary files

**Review by AV29 — APPROVED** [review]

**Pull request merged**